node/src/manager/commands/index.rs

```diff
@@ -34,12 +35,23 @@ pub async fn create(
     search: DeploymentSearch,
     entity_name: &str,
     field_names: Vec<String>,
-    index_method: String,
+    index_method: Option<String>,
     after: Option<i32>,
 ) -> Result<(), anyhow::Error> {
     validate_fields(&field_names)?;
     let deployment_locator = search.locate_unique(&pool)?;
     println!("Index creation started. Please wait.");
+
+    // If the fields contain the block range column, we use GIN
+    // indexes. Otherwise we default to B-tree indexes.
+    let index_method = index_method.unwrap_or_else(|| {
+        if field_names.contains(&BLOCK_RANGE_COLUMN.to_string()) {
+            "gist".to_string()
+        } else {
+            "btree".to_string()
+        }
+    });
```

Comment:
This is really minor, but do we need all the `to_string()` here? (If so, that's ok)